arviz/plots/compareplot.py

```diff
@@ -53,26 +53,26 @@ def plot_compare(
         If True (default) ensure the best model is used as reference.
     legend : bool
         Add legend to figure. By default True.
-    figsize : tuple, optional
+    figsize : (float, float), optional
         If None, size is (6, num of models) inches
-    title : bool:
+    title : bool
         Show a tittle with a description of how to interpret the plot. Defaults to True.
-    textsize: float
+    textsize : float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
         on ``figsize``.
-    labeller : labeller instance, optional
+    labeller : Labeller instance, optional
         Class providing the method ``model_name_to_str`` to generate the labels in
         the plot.
         Read the :ref:`label_guide` for more details and usage examples.
     plot_kwargs : dict, optional
         Optional arguments for plot elements. Currently accepts 'color_ic',
         'marker_ic', 'color_insample_dev', 'marker_insample_dev', 'color_dse',
         'marker_dse', 'ls_min_ic' 'color_ls_min_ic',  'fontsize'
-    ax: axes, optional
-        Matplotlib axes or bokeh figures.
-    backend: str, optional
+    ax : axes, optional
+        Matplotlib _axes or bokeh _figure.

```

Comment:
```suggestion
    ax : matplotlib_axes or bokeh_figure, optional
        Matplotlib axes or bokeh figure on which to plot. If ``None``, one will be generated.
```

This was modifying the parameter description section instead of the parameter type section. The aliases only work and can only be used in the parameter type section.